classes/page_group.php

```diff
@@ -61,6 +61,12 @@ protected function set_fuzzydurationcounts(array $data) {
      */
     public static function record_fuzzy_counts(profile $profile) {
 
```

Comment:
Can we remove the extra whitespace here.

Reply:
done